### DISCUSSION TITLE
Increasing number of data points for datarate meaurement

### DISCUSSION THREAD

**Discussion by PhilippComet** [inet-framework/inet#904] [General]
Hi everybody,

I would like to increase the datapoints for measuring the ingoing and outgoing datarate at the queue module in the Time Aware Shaper. 
For the throughput, I saw in the showcase example that there is a default interval for the sliding window of 0.1s. I assume there is a simliar sliding window the the datarate measurement. How can this sliding window be decreased in length to get more samples/datapoints in the .vec file?

Best regards,
Philipp

**Comment by levy** [comment]
You can change the result filter parameter from the INI file.
For example,

**.eth[*].queue.incomingDataRate.throughput.interval = 10ms

In general, you can configure result filters as
<module-path>.<statistic-name>.<result-filter-name>.<parameter-name>

**Reply by PhilippComet** [reply]
Thank you for the response. I will try that 👍🏻
